core/domain/activity_validators_test.py

```diff
@@ -0,0 +1,160 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for core.domain.activity_validators."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+
+from constants import constants
+from core.domain import collection_domain
+from core.domain import collection_services
+from core.domain import exp_domain
+from core.domain import exp_services
+from core.domain import prod_validation_jobs_one_off
+from core.domain import user_services
+from core.platform import models
+from core.tests import test_utils
+
+(activity_models,) = models.Registry.import_models([models.NAMES.activity])
+
+
+class ActivityReferencesModelValidatorTests(test_utils.AuditJobsTestBase):
+
+    def setUp(self):
+        super(ActivityReferencesModelValidatorTests, self).setUp()
+
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.owner = user_services.UserActionsInfo(self.owner_id)
+
+        exploration = exp_domain.Exploration.create_default_exploration(
+            '1exp', title='title', category='category')
+
+        exp_services.save_new_exploration(self.owner_id, exploration)
+
+        collection = collection_domain.Collection.create_default_collection(
+            '1col', title='title', category='category')
+
+        collection_services.save_new_collection(self.owner_id, collection)
+
+        self.model_instance = (
+            activity_models.ActivityReferencesModel.get_or_create('featured'))
```

Comment:
Considering this as s test setup block should we use the existing feconf constant for this value (featured)?

Reply:
Yes, done.